src/pages/services.tsx

```diff
@@ -19,6 +52,116 @@ export default ServicesPage;
 
 export const ServicesPageQuery = graphql`
   query ($language: String!) {
+    contentfulPage(slug: { eq: "services" }, node_locale: { eq: $language }) {
+      title
+      description {
+        description
+      }
+    }
+
+    contentfulLeadbox(
+      slug: { eq: "services-leadbox" }
+      node_locale: { eq: $language }
+    ) {
+      title
+      illustration
+      contact {
+        headline
+        title
+        email
+      }
+    }
+
+    servicesHeader: contentfulSectionHeader(
+      slug: { eq: "services-introduction" }
+      node_locale: { eq: $language }
+    ) {
+      kicker
+      headline
+      paragraphs {
+        paragraph {
+          paragraph
+        }
+      }
+    }
```

Comment:
That approach to fetch individual content models of a specific type is pragmatic. I see where you're coming from but there is plenty of space to make this better (we as a team 🙏) once we finally landed this ugly big PR